transportmiddleware/interface_test.go

```diff
@@ -0,0 +1,85 @@
+package transportmiddleware
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/yarpc/yab/transport"
```

Comment:
nit: Import group ordering. It should be,

- standard library
- local imports (packages of the yab repo)
- external imports